sunpy/net/dataretriever/sources/noaa.py

```diff
@@ -137,10 +149,10 @@ class SRSClient(GenericClient):
     Results from 1 Provider:
     <BLANKLINE>
     2 Results from the SRSClient:
-         Start Time           End Time      Instrument Physobs Source Provider
-    ------------------- ------------------- ---------- ------- ------ --------
-    2016-01-01 00:00:00 2016-12-31 23:59:59       SOON     SRS   SWPC     NOAA
-    2016-01-01 00:00:00 2016-12-31 23:59:59       SOON     SRS   SWPC     NOAA
+           Start Time               End Time        Instrument ... Source Provider
+    ----------------------- ----------------------- ---------- ... ------ --------
+    2016-01-01 00:00:00.000 2016-01-02 00:00:00.000       SOON ...   SWPC     NOAA
+    2016-01-01 00:00:00.000 2016-01-02 00:00:00.000       SOON ...   SWPC     NOAA
```

Comment:
Is it okay that the time has changed here?

Reply:
Huh, I thought that this bug was already fixed in #4600

Neither the removed or the added lines are correct here though :thinking:

Reply:
Is this something we need to tackle?

Reply:
I sorta want to punt this to it's own issue?

Reply:
Yeah, move it outside of this PR.

Reply:
https://github.com/sunpy/sunpy/issues/4922